### PR TITLE
feat: implement random seed on flux1 model

### PR DESCRIPTION
**Pull request by hugohonda**

### CODE REVIEW

**Comment by hrnn** [comment]
overall it LGTM, QQ. shall we update https://github.com/landing-ai/vision-agent-tools/blob/main/docs/flux1.md?

**Comment by hugohonda** [comment]
> overall it LGTM, QQ. shall we update https://github.com/landing-ai/vision-agent-tools/blob/main/docs/flux1.md?

Agree! Thank you for remembering it. Update the docs

**Review by hrnn — APPROVED** [review]
:shipit:

File: docs/flux1.md [hunk 1]
```diff
@@ -1,44 +1,61 @@
...
+flux_model = Flux1()
 
-# Generate an image from the prompt
 generated_image = flux_model(
-    task=Flux1Task.IMAGE_GENERATION,  # Image Generation Task
-    prompt=prompt,
+    task=Flux1Task.IMAGE_GENERATION,
+    prompt="A purple car in a futuristic cityscape",
+    height=1024,
+    width=1024,
```
Comment (Dayof) [inline docs/flux1.md:34]: it's a common good practice to add height and width together, e.g. numpy shape order is (H, W, C)

**Review by Dayof — APPROVED** [review]

**Comment by hugohonda** [comment]
Tests:

image

**Pull request merged**